scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/utils/KeyReaderWriter.scala

```diff
@@ -16,19 +16,39 @@
 
 package com.intel.analytics.bigdl.ppml.utils
 import java.io.PrintWriter
+import java.net.URI
+
+import org.apache.hadoop.conf.Configuration
+import org.apache.hadoop.fs.{FileSystem, Path}
+import org.apache.hadoop.io.IOUtils
+
 import scala.io.Source
 
 class KeyReaderWriter {
 
-  def writeKeyToFile(encryptedKeyPath: String, encryptedKeyContent: String): PrintWriter = {
-    new PrintWriter(encryptedKeyPath) {
-      write(encryptedKeyContent); close
-    }
+  def writeKeyToFile(encryptedKeyPath: String, encryptedKeyContent: String,
+                     config: Configuration = null): Unit = {
+//    new PrintWriter(encryptedKeyPath) {
+//      write(encryptedKeyContent); close
+//    }
+    val hadoopConfig = if (config != null) config else new Configuration()
+    val fs: FileSystem = FileSystem.get(new URI(encryptedKeyPath), hadoopConfig)
+    val outputStream = fs.create(new Path(encryptedKeyPath))
+    println(s"write key to file: ${encryptedKeyContent}")
```

Comment:
remove the `println` if not necessary.

Reply:
Will fix in https://github.com/intel-analytics/BigDL/pull/4783